client/finality-grandpa/src/lib.rs

```diff
@@ -866,7 +873,6 @@ where
 		debug!(target: "afg", "{}: Starting new voter with set ID {}", self.env.config.name(), self.env.set_id);
 
 		let authority_id = is_voter(&self.env.voters, &self.env.config.keystore)
```

Comment:
We can update `is_voter` below to take `Option<&BareCryptoStorePtr>` and pass `self.env.config.keystore.as_ref()` here instead.